cartridge/argparse.lua

```diff
@@ -353,6 +353,10 @@ local function parse_file(filename, search_name)
     local ret = {}
     for _, section_name in ipairs(section_names) do
         local content = file_sections[section_name] or {}
+        if type(content) ~= 'table' then
+            return nil, ParseConfigError:new('Invalid content: options should be '..
+                'in format `name: value`')
```

Comment:
I would elaborate on the error message here. Here is some thoughts:
- it is more informative to say that a content must be a map or table with the format of `key: value`
- also it would be point out in which file and section the error occured

For example, for my reproducer the error message could look like this:

```
Invalid content found in file "cfg.yml" in section "default": expected a table of key-value pairs, got string
```